jenkins.el

```diff
@@ -52,6 +52,7 @@
     (define-key keymap (kbd "1") 'jenkins-job-details-toggle)
     (define-key keymap (kbd "b") 'jenkins--call-build-job-from-job-screen)
     (define-key keymap (kbd "v") 'jenkins--visit-job-from-job-screen)
+    (define-key keymap (kbd "c") 'jenkins--show-console-output-from-job-screen)
```

Comment:
I'd like to '$' here. You know it is used in magit/prodigy packages for likely the same purposes.